flagparse/flags.go

```diff
@@ -171,7 +172,7 @@ func initConfigValues(typeOfConfig reflect.Type, valueOfConfig reflect.Value, fl
 		case bool:
 			defaultBoolValue, err := strconv.ParseBool(defaultValue)
 			if err != nil {
-				log.Println(fmt.Sprintf("Error parsing boolean argument %v with value %v", fieldType.Name, defaultValue))
+				log.Printf("Error parsing boolean argument %v with value %v", fieldType.Name, defaultValue)
```

Comment:
A few points here:
* Is it possible at all? The default values should be valid.
* The message should say that it's the default value.
* Is it enough to have only log?

Reply:
This code handles incorrect definition(s) of flag default values. Maybe it should return an error/panic?

Reply:
Separate issue has been posted - [#25].